### ISSUE TITLE
Use `duration` (and any other useful AV properties?) in IIIF v3 manifests

### ISSUE THREAD

**Issue by cbeer** [sul-dlss/purl#159]
- [ ] find some fixture objects with `audioData` or `videoData` exposed
- [ ] add `duration` property to media content resources (http://prezi3.iiif.io/api/presentation/3.0/#duration)

**Comment by jmartin-sul** [comment]
we're moving this back to backlog since the following concerns made this seem not particularly useful at the moment:
1. we don't seem to have any objects in stage or prod with `audioData` or `videoData` in the `contentMetadata` data stream, and we're not very confident that we'll be producing objects like that any time soon.
1. if we did have duration in `contentMetadata`, it'd be in the ISO duration format.  however, the IIIF spec calls for duration to be ["given in seconds as a non-negative floating point number"](http://prezi3.iiif.io/api/presentation/3.0/#duration).  so we'd have to convert, which shouldn't be difficult.  but that might indicate that either our `contentMetadata` or the IIIF spec should switch duration format.
1. we're not sure at the moment how duration info would practically be used by consumers of the manifest.  for instance, sul-embed shows duration for media it streams based on the metadata provided in the actual video stream.  but it does seem like it could be useful for, e.g. showing what the duration of a video or audio file would be without having to load the file at all.

in standup this morning, it seemed like numbers 2 & 3 could be turned into useful questions for the IIIF community, since the version 3 spec is still in flux.  @tingulfsen was volunteered to pose those questions on an upcoming IIIF call.

number 2 could also be a useful question for SMPL: we're expecting an ISO duration in `contentMetadata`, because that's what they said they'd produce when we asked them about it while working on media streaming last summer.  we weren't sure in standup this morning why they'd prefer to produce that.  maybe they could use seconds?  could @blalbrit or @LynnMcRae pass that question along.

in poking around yesterday, i did notice that lots of objects have duration in `technicalMetadata`, but @blalbrit said that we should not use that duration info (FWIW, it was usually specified multiple times in the datastream, in different formats, e.g. minutes/seconds and milliseconds, with all values being within roughly a second of one another).

**Comment by LynnMcRae** [comment]
Just for some history, the ISO 8601 specification for durations was suggested by SMPL and chosen in our model mainly on the strength of it being a standards-based way of expressing elapsed time.  It covers timespans that might be days or months or years. 

But IIIF is a standard as well, and at some point this information will no longer be derived from public XML but rather from whatever linked data assertions are present in the SDR3 model, and those assertions will also name the manner of encoding used to express the interval.

Basically it means we could feel free to adopt seconds for contentMetadata if implemented in the short term, as it will be translated appropriately into the new data model when the time comes.

**Comment by cbeer** [comment]
Here is the current sul-embed implementation of `duration`: https://github.com/sul-dlss/sul-embed/commit/6b2338a9a38db4461928582bc1630254f9d0541d
